### DISCUSSION TITLE
VLC supports?

### DISCUSSION THREAD

**Discussion by apovetkin** [unoplatform/uno#7926] [Q&A]
Hi guys,

Could you tell me if Uno Platform supports VLC?

Any example very appreciated.

Thanks,
Alexander

**Comment by jeromelaban** [comment]
Can you tell which platform you are interested in ?

**Reply by apovetkin** [reply]
Windows and ANDROID

**Reply by jeromelaban** [reply]
See [this package](https://www.nuget.org/packages/LibVLCSharp.Uno/) for the list of supported platforms.

**Reply by apovetkin** [reply]
Nice!!!

**Reply by AbelWu1** [reply]
@jeromelaban Hi，I want to use vlc for linux , I use the uno platform and write it in C #, Could you give me some examples? Thanks.
